samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Events.razor

```diff
@@ -147,9 +147,13 @@
 
 @code {
 
-    private void OnMapClick(ClickEvent clickEvent)
+    private async Task OnMapClick(ClickEvent clickEvent)
     {
         _clickJson = BuildViewableJson(clickEvent);
+        ScreenPoint screenPoint = await _view.ToScreen(clickEvent.MapPoint);
```

Comment:
excited about putting this to use.  I will need to look some at the docs to see more if the same conversion methods will apply the same when drawing lines, polygons etc when multiple points are applied to the graphic, vs singular points

Reply:
Even though there is a `Point` referenced here, it is not necessarily a graphic, and the conversions would not be available for a line or polygon, but you could for example convert a single point _from_ a line/polygon.